package.json

```diff
@@ -3,9 +3,9 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "dev": "next dev",
+    "dev": "NODE_OPTIONS='-r @newrelic/next' next dev",
     "build": "next build",
-    "start": "next start",
+    "start": "NODE_OPTIONS='-r @newrelic/next' next start",
```

Comment:
We don't want to include new relic locally so it should be okay to remove from the "dev" script.

For the "start" script, I think we should keep the original `"start": "next start"` command since this gets called automatically on Vercel and I don't think we want to track new relic there. A new one can be added: `"start:newrelic": "NODE_OPTIONS='-r @newrelic/next' next start",` . This new one should be called in the Dockerfile for AWS.

Reply:
👍